robot_skills/src/robot_skills/head.py

```diff
@@ -1,21 +1,100 @@
 #! /usr/bin/env python
+import math
+from collections import namedtuple
+
 import rospy
 from threading import Condition
 from geometry_msgs.msg import PointStamped
 from head_ref.msg import HeadReferenceAction, HeadReferenceGoal
+from std_srvs.srv import Empty
+from std_msgs.msg import ColorRGBA
+from visualization_msgs.msg import Marker, MarkerArray
+from image_recognition_msgs.srv import Annotate, Recognize, RecognizeResponse, GetPersons
+from image_recognition_msgs.msg import Annotation
 
-from sensor_msgs.msg import Image
+from sensor_msgs.msg import Image, RegionOfInterest
 from robot_part import RobotPart
 
+# TU/e
+from rgbd.srv import Project2DTo3D, Project2DTo3DRequest
+
 from .util import msg_constructors as msgs
 from .util.kdl_conversions import kdlVectorStampedToPointStamped, VectorStamped
 
 
+WavingResult = namedtuple('WavingResult', ['side', 'roi'])
```

Comment:
The skeleton-related stuff makes more sense in a robot.perception-part, or eyes if you will.